<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkIGTRecorderControls</class>
 <widget class="QWidget" name="QmitkIGTRecorderControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkIGTExample</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="m_DeviceConfigGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Select Tracking Device</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_UpdateTrajectory">
     <property name="text">
      <string>Update Trajectory</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_UpdateRecorder">
     <property name="text">
      <string>Update Recorder</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_StartRecording">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start Recording</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_PauseRecording">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Pause Recording</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_StopRecording">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop Recording</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_StartReplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replay recorded file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_StopReplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop Replaying</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">mitkPropertyList.h</include>
  <include location="local">QmitkPlotWidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
